server/src/main/java/org/hyperskill/community/flashcards/card/ExampleDataInitializer.java

```diff
@@ -0,0 +1,130 @@
+
+/**
+ * Initializes the 'example' database with sample cards.
+ */
+@Component
+@Slf4j
+public class ExampleDataInitializer implements CommandLineRunner {
+    private final MongoTemplate mongoTemplate;
+
+    public ExampleDataInitializer(@Qualifier("example") MongoTemplate mongoTemplate) {
+        this.mongoTemplate = mongoTemplate;
+    }
+
+    @PostConstruct
+    public void init() {
+        List<Card> cards = List.of(
+                QuestionAndAnswerCard.builder()
+                        .author("test@test.com")
+                        .title("card 1")
+                        .category("Math")
+                        .isPublic(true)
+                        .question("Calculate 2 + 2 = ?")
+                        .answer("4")
+                        .build(),
+                SingleChoiceQuiz.builder()
+                        .author("test@test.com")
+                        .title("card 2")
+                        .category("Math")
+                        .isPublic(true)
+                        .question("Solve the equation: 2x + 3 = 9")
+                        .options(List.of("2", "5", "6", "3"))
+                        .correctOption(3)
+                        .build(),
+                MultipleChoiceQuiz.builder()
+                        .author("test@test.com")
+                        .title("card 3")
+                        .category("Math")
+                        .isPublic(true)
+                        .question("Select all correct statements about the equation: 2x + 3 = 9")
+                        .options(List.of("It's a square equation", "It's a linear equation", "It doesn't have a root", "It's root is 3"))
+                        .correctOptions(List.of(1, 3))
+                        .build(),
+                QuestionAndAnswerCard.builder()
+                        .author("test@test.com")
+                        .title("card 4")
+                        .category("Geography")
+                        .isPublic(true)
+                        .question("Capital of Spain")
+                        .answer("Madrid")
+                        .build(),
+                SingleChoiceQuiz.builder()
+                        .author("test@test.com")
+                        .title("card 5")
+                        .category("Chemistry")
+                        .isPublic(true)
+                        .question("Which of the elements has the lowest atomic weight?")
+                        .options(List.of("Au", "Fe", "Li", "Pt"))
+                        .correctOption(2)
+                        .build()
+        );
+
+        log.info("Dropping any existing collections in 'example' database...");
+        var collections = mongoTemplate.getDb().listCollectionNames();
+
+        log.info("Inserting sample data to 'example' database...");
+        cards.forEach(card -> {
+            var category = card.getCategory();
+            mongoTemplate.insert(card, category);
+        });
+    }
```

Comment:
Ah you thought of a full example **database**? I gathered, we'd put all cards into the _collection_ named "example". (see comment at MongoConfiguration)

Reply:
See above :arrow_up: